test/index.js

```diff
@@ -379,6 +381,54 @@ describe('Memory', function () {
         done();
     });
 
+    describe('stopping after set', function () {
```

Comment:
Yes, a single `it`. Your method patches aren't significant. Just make sure to restore the old behavior before making any assertions. Once the test is fixed up, this should be good to go.